configure

```diff
@@ -138,6 +138,38 @@ for dep in $IGNORE; do
     fi
 done
 
+echo "uftrace detected system features:"
+
+print_feature()
+{
+    item=$1
+    file=$2
+    description=$3
+
+    if [ -t 1 ]; then
+        # use colored output only when stdout is tty
+        if [ -f ${srcdir}/check-deps/${file} ]; then
+            onoff="\033[32mon \033[0m"
+        else
+            onoff="\033[91mOFF\033[0m"
+        fi
+    else
+        if [ -f ${srcdir}/check-deps/${file} ]; then
+            onoff="on "
+        else
+            onoff="OFF"
+        fi
+    fi
+    printf "...%15s: [ ${onoff} ] - %s\n" "${item}" "${description}"
+}
+
+printf "...%15s: %s\n" "prefix" "${prefix}"
+print_feature "libelf" "have_libelf" "more flexible ELF data handling"
+print_feature "libdw" "have_libdw" "DWARF debug info support (for argument, location, ...)"
```

Comment:
I'm sorry but it seems bit long so let's removed the parenthesis part.

Reply:
No problem.